src/snovault/elasticsearch/tests/test_qintegration_simple.py

```diff
@@ -0,0 +1,203 @@
+"""Tests the simple queue with a mocked indexer"""
```

Comment:
Rename file to queue_integration...?

Reply:
made it indexer test since that's what it's doing.